recipes/bsp/grub2.yaml

```diff
@@ -0,0 +1,84 @@
+inherit: [autotools, autoconf]
+
+metaEnvironment:
+    PKG_VERSION: "2.12"
+
+Config:
+    GRUB2_BOOTSTRAP:
+        help: Run bootstrap in the sources. This is only needed if a git repo is used
+        type: bool
+    GRUB2_TARGET:
+        help: "target selection. (i386, x86_64,..). Default: x86_64"
+        type: str
+    GRUB2_PLATFORM:
+        help: "Platform selection. (efi, pc, ...). Default: efi"
+        type: str
```

Comment:
Maybe use a `choice`?

Reply:
Look like there are [many choices](https://git.savannah.gnu.org/cgit/grub.git/tree/configure.ac#n172) Not sure if this is worth it?

Reply:
No, I think you're right. This is too generic.